com.github.kagami.boram.yml

```diff
@@ -0,0 +1,243 @@
+          - name: x265
+            buildsystem: simple
+            build-options:
+              arch:
+                aarch64:
+                  cxxflags: '-fPIC'
+            build-commands:
+              - mkdir source/build-8 source/build-10 source/build-12
+              - >-
+                cd source/build-12 && cmake ..
+                -DCMAKE_INSTALL_PREFIX=${FLATPAK_DEST} -DHIGH_BIT_DEPTH='TRUE'
+                -DMAIN12='TRUE' -DEXPORT_C_API='FALSE' -DENABLE_CLI='FALSE'
+                -DENABLE_SHARED='FALSE'
+              - cd source/build-12 && make
+              - >-
+                cd source/build-10 && cmake ..
+                -DCMAKE_INSTALL_PREFIX=${FLATPAK_DEST} -DHIGH_BIT_DEPTH='TRUE'
+                -DEXPORT_C_API='FALSE' -DENABLE_CLI='FALSE' -DENABLE_SHARED='FALSE'
+              - cd source/build-10 && make
+              - ln -s ../build-10/libx265.a source/build-8/libx265_main10.a
+              - ln -s ../build-12/libx265.a source/build-8/libx265_main12.a
+              - >-
+                cd source/build-8 && cmake .. -DCMAKE_INSTALL_PREFIX=${FLATPAK_DEST}
+                -DENABLE_SHARED='TRUE' -DENABLE_CLI='FALSE'
+                -DENABLE_HDR10_PLUS='TRUE' -DEXTRA_LIB='x265_main10.a;x265_main12.a'
+                -DEXTRA_LINK_FLAGS='-L.' -DLINKED_10BIT='TRUE' -DLINKED_12BIT='TRUE'
+              - cd source/build-8 && make
+              - cd source/build-8 && make install
+            sources:
+              - type: archive
+                url: >-
+                  https://bitbucket.org/multicoreware/x265_git/downloads/x265_3.3.tar.gz
+                sha256: c6d744a87eda55560da715f56f878640554ddc06e2d0fcbd822fa330affc22cc
+          - name: fdkaac
+            config-opts:
+              - '--disable-static'
+            sources:
+              - type: archive
+                url: https://github.com/mstorsjo/fdk-aac/archive/v2.0.1.tar.gz
+                sha256: a4142815d8d52d0e798212a5adea54ecf42bcd4eec8092b37a8cb615ace91dc6
+          - name: lame
+            cleanup:
+              - /share/doc
+            config-opts:
+              - '--enable-shared'
+              - '--disable-static'
+              - '--disable-frontend'
+              - '--enable-nasm'
+            sources:
+              - type: archive
+                url: https://downloads.sourceforge.net/lame/lame-3.100.tar.gz
+                sha256: ddfe36cab873794038ae2c1210557ad34857a4b6bdc515785d1da9e175b1da1e
+                
+  - name: boram
+    buildsystem: simple
+    build-commands:
+      - install -D boram.sh /app/bin/Boram
+      - install -d /app/lib/Boram
+      - install -Dm644 -t /app/share/applications/ com.github.kagami.boram.desktop
+      - install -Dm644 resources/app/icon-big.png /app/share/icons/com.github.kagami.boram.png
+      - install -Dm644 -t /app/share/metainfo/ com.github.kagami.boram.metainfo.xml
+      - cp -r * /app/lib/Boram
+    only-arches:
+      - x86_64
```

Comment:
this will no prevent the flatpak to be build on aarch64. It will merely skip this module.

Why not building it from source?

Reply:
I had problems compiling it locally so I'm not very confident I can get it to compile as a Flatpak. The repos don't really go into that.

Is there a way to prevent it from being build on aarch64 without compiling from source?

Reply:
I'd rather have a build from source 

but if you must https://github.com/flathub/flathub/wiki/App-Maintenance#only-build-on-x86_64